unused-images-crawler/index.js

```diff
@@ -0,0 +1,138 @@
+require("dotenv").config();
+// BEGIN: Configuration
+
+const oauthToken = process.env.OAUTH_TOKEN;
+const spaceId = parseInt(process.env.SPACE_ID);
+let folderToAssign = parseInt(process.env.FOLDER_TO_ASSIGN);
```

Comment:
This is never used in the code as by default you are checking if there's a folder called "unused-assets" and then you either override this `folderToAssign` with its ID or you create a new folder and you use its ID. 

I would suggest changing the code so you can check first if the folder set in this variable exists and then eventually create one. 

I would suggest looking at how this was used in the code example in our internal ticket as that can help.

Reply:
You're right! But as the folderToAssign is used in several places I have just instantiated it here...

Reply:
I think it was better the initial approach with the customer folder plus the fallback, you just have to try to see if the folder set by the user exists and if it doesn't you can check if the fallback exists and if that doesn't exist either you can then create it.